src/Engine/ProtoCore/DSASM/Mirror/ExecutionMirror.cs

```diff
@@ -295,9 +295,9 @@ private int GetSymbolIndex(string name, out int ci, ref int block, out SymbolNod
 
             if (runtimeCore.DebugProps.DebugStackFrameContains(DebugProperties.StackFrameFlagOptions.FepRun))
             {
-                ci = runtimeCore.watchClassScope = rmem.CurrentStackFrame.ClassScope;
-                functionIndex = rmem.CurrentStackFrame.FunctionScope;
-                functionBlock = rmem.CurrentStackFrame.FunctionBlock;
```

Comment:
Do we still need these properties on `StackFrame` or can they be deprecated/removed?

Reply:
I do not think so.  The main concern was the `CurrentStackFrame` object.  It was always generating a new `StackFrame` object which was happening millions of times.  If you already have a `StackFrame` object these properties are fine.  I am only shortcutting the instantiation for the `CurrentStackFrame` wrapper class.